src/screens/HyperSwitch/RoutingRevamp/AdvancedRoutingUIUtils.res

```diff
@@ -283,11 +283,39 @@ let metaInput = (id, keyType) =>
 
 module FieldInp = {
   @react.component
-  let make = (~ops, ~prefix, ~onChangeMethod) => {
+  let make = (~prefix, ~onChangeMethod) => {
     let field = ReactFinalForm.useField(`${prefix}.lhs`).input
     let op = ReactFinalForm.useField(`${prefix}.comparison`).input
     let val = ReactFinalForm.useField(`${prefix}.value.value`).input
 
+    let keyDescriptionMapper = Window.getDescriptionCategory()->MapTypes.changeType
+    let convertedValue = LogicUtils.convertMapObjectToDict(keyDescriptionMapper)
```

Comment:
can we encapsulate the `keyDescriptionMapper` and `convertedValue` under a memo function and return a single value from the memo

Reply:
updated